plugins/rabbitmq_logs.yaml

```diff
@@ -0,0 +1,34 @@
+version: 0.0.1
+title: RabbitMQ
+description: Log parser for RabbitMQ
+parameters:
+  - name: daemon_log_path
+    type: string
+    default: "/var/log/rabbitmq/rabbit@*.log"
```

Comment:
Something I tried to do with more recent stanza plugins, was to use `strings` instead of `string` for most input files. I think generally, globbing is fine but sometimes we end up in a situation where the user needs to specify multiple strings explicitly. 

With Rabbit, it might not matter but I think the cost / complexity associated with strings is not too high.

I think this would be worth standardizing on this early, if we agree.

Reply:
That's an interesting take we could do this and I don't think there would be any issues as this maps to the include paths

Reply:
Stanza plugins is somewhat inconsistent here, so I think we can standardize on it early.

Reply:
I agree completely that we should standardize on ability to include multiple strings. Then, if appropriate, we can make exceptions the other way if there is good reason to do so.